### ISSUE TITLE
"Error loading property" with version 1.24.0 when connecting to Azure Government Storage Accounts

### ISSUE THREAD

**Issue by johnsblevins** [microsoft/AzureStorageExplorer#5726]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.24.0

### Architecture

x64

### Storage Explorer Build Number

20220503.4

### Platform

Windows

### OS Version

Windows 10

### Bug Description

"Error loading property" with version 1.24.0 when connecting to Azure Government Storage Accounts.  Occurs after adding Subscription, all storage accounts in Azure Government subscriptions fail to enumerate Blob Containers or Blobs.  Error Loading Property is display in Properties section.  This only occurs when I run Storage Explorer from a Virtual Machine in Azure Commercial or Azure Government.  When connecting using Storage Explorer from my personal workstation at home or office it works as expected.

### Steps to Reproduce

From Virtual Machine in Azure Commercial or Azure Government
1. Launch Azure Storage Explorer
2. Add Azure Government Subscription
3. Drill down into subscription, storage account to container level

### Actual Experience

![image](https://user-images.githubusercontent.com/17452681/168386661-9355a8be-b0a9-4f1a-b9c1-212d82beefa0.png)

### Expected Experience

Expect to see blob container and blobs enumerated.

**Comment by MRayermannMSFT** [comment]
@johnsblevins yes this was an issue with 1.24.0. We just shipped a hotfix on Friday that addresses it: https://github.com/microsoft/AzureStorageExplorer/releases/tag/v1.24.1

Please download and use that version. Thanks.

**Comment by MRayermannMSFT** [comment]
Ok...closing as duplicate of https://github.com/microsoft/AzureStorageExplorer/issues/5699
